packages/zoe/src/types.js

```diff
@@ -237,80 +204,55 @@
  * any newAllocations have keywords that are not in
  * sparseKeywords.
  *
- * @param  {OfferHandle[]} offerHandles An array of offerHandles
- * @param  {AmountKeywordRecord[]} newAllocations An
- * array of amountKeywordRecords  - objects with keyword keys
- * and amount values, with one keywordRecord per offerHandle.
- * @returns {void}
- *
- * @callback Complete
- * The contract can "complete" an offer to remove it from the
- * ongoing contract and resolve the player's payouts (either
- * winnings or refunds). Because Zoe only allows for
- * reallocations that conserve rights and are 'offer-safe', we
- * don't need to do those checks at this step and can assume
- * that the invariants hold.
- * @param  {OfferHandle[]} offerHandles - an array of offerHandles
+ * @param  {StagedSeat} stagedSeat
+ * @param {StagedSeat} stagedSeat
+ * @param {StagedSeat=} stagedSeat
+ * @param {StagedSeat=} stagedSeat
+ * @param {StagedSeat=} stagedSeat
```

Comment:
I don't understand this syntax. OT1H it seems to say we can have at most 5 such arguments. OTOH, it gives all the parameter variables the same name. Both seem wrong.

Reply:
If anyone knows a better way to get variadic functions recognized by typescript, let me know, but all suggestions so far did not work entirely. This is the best so far in that it will only show errors for the contract developer if they try to reallocate over more seats than the params.

Reply:
Attn @michaelfig